examples/local_sgd/resnet/train.py

```diff
@@ -169,24 +193,21 @@ def get_device_num():
     device_num = fluid.core.get_cuda_device_count()
     return device_num
 
-def get_local_sgd_steps(passid, local_sgd_steps, local_sgd_warmup, lsgd_warmup_strategy):
-    offset = passid - local_sgd_warmup
-    if offset < 0:
-        return 1
-    if lsgd_warmup_strategy == 1:
-        warm_up = [2**i for i in range(local_sgd_steps) if 2**i <=local_sgd_steps]
-    elif lsgd_warmup_strategy == 2:
-        warm_up = [local_sgd_steps]
-    elif lsgd_warmup_strategy == 3:
-        warm_up = [2*i for i in range(local_sgd_steps) if 2*i <=local_sgd_steps]
-        warm_up[0] = 1
-
-    warm_size = len(warm_up)
-    if offset >= warm_size:
-        return local_sgd_steps
-    else:
-        return warm_up[offset]
-     
+def AdaCom(ini_loss, ini_lr, cur_loss, cur_lr, base_step, pre_step):
```

Comment:
inconsistent format of function name

Reply:
And you'd better put the reference paper for this algorithmn

Reply:
Done! add reference: https://arxiv.org/pdf/1810.08313.pdf